<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorldmapEditor</class>
 <widget class="QWidget" name="WorldmapEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QToolBox" name="toolBox">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page_3">
         <attribute name="label">
          <string>Map</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Tile Pixel Size</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="2" column="1">
              <widget class="QSpinBox" name="tileSizeY"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Y</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="tileSizeX"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>X</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Amount of Tiles</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="2" column="1">
              <widget class="QSpinBox" name="mapSizeY"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Y</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>X</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="mapSizeX"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>305</width>
           <height>276</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Cities</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="buttonAddCity">
            <property name="text">
             <string>Add City</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="cityList"/>
          </item>
          <item>
           <widget class="QFrame" name="cityFrame">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="2" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Position Y</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Visible</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QDoubleSpinBox" name="cityPosY"/>
             </item>
             <item row="3" column="1">
              <widget class="QDoubleSpinBox" name="cityRadius"/>
             </item>
             <item row="4" column="1">
              <widget class="QCheckBox" name="cityVisible">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Position X</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0" colspan="2">
              <widget class="QPushButton" name="buttonDeleteCity">
               <property name="text">
                <string>Delete City</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDoubleSpinBox" name="cityPosX"/>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Radius</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>288</width>
           <height>388</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Terrain</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="terrainSelectAll">
            <property name="text">
             <string>Select All Cases</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="terrainDiscardSelection">
            <property name="text">
             <string>Discard Selection</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_2">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Movement speed</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
             </item>
             <item row="2" column="0" colspan="2">
              <widget class="QFrame" name="frame_3">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QGridLayout" name="gridLayout_3">
                <item row="1" column="0">
                 <widget class="QPushButton" name="encounterAddMap">
                  <property name="text">
                   <string>Add</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QPushButton" name="encounterAddType">
                  <property name="text">
                   <string>Add</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QPushButton" name="encounterDelMap">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Delete</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1">
                 <widget class="QPushButton" name="encounterDelType">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Delete</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="0" colspan="2">
                 <widget class="QListWidget" name="encounterTypes"/>
                </item>
                <item row="2" column="0" colspan="2">
                 <widget class="QListWidget" name="encounterMapList"/>
                </item>
                <item row="3" column="0" colspan="2">
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>Encounter types</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0" colspan="2">
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>Maps</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="0" colspan="2">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Random Encounters:</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
